tests/misc/test_basic_without_ssh.py

```diff
@@ -1,4 +1,6 @@
+import logging
 import pytest
+
```

Comment:
These header code style changes should maybe belong to another PR?

Reply:
I added an import as well i don't think making a specific PR for this is relevant since they'd be no functional changes at all so a revert would have no consequences.

Reply:
Ok